XU-TechnicalPaper.md

```diff
@@ -25,163 +25,176 @@
 
-In general, all BIP 199 compatible payment channels can be supported directly, others might need to find a way to implement HTLC swaps via smart contracts or similar. The payment channel standard on bitcoin and litecoin is called ‘[lightning](https://github.com/lightningnetwork/lnd)’ and compatible implementations are maintained by the companies [ACINQ](https://github.com/ACINQ/eclair), [Lightning Labs](https://lightning.engineering/) and [Blockstream](https://blockstream.com/). Compatibility was only achieved [recently](https://medium.com/@lightning_network/lightning-protocol-1-0-compatibility-achieved-f9d22b7b19c4), implementation of such is still ongoing. Ethereum’s payment channel technology is called the ‘[Raiden Network](https://raiden.network/)’ and is maintained by the company [Brainbot](http://www.brainbot.com/). **XUDs are a subgraph of the lightning & raiden network, which means it is ensured that the XUD implementation remains fully compatible with the [lightning protocol specifications (BOLT)](https://github.com/lightningnetwork/lightning-rfc/blob/master/00-introduction.md) and also with the raiden network to support the growth and adoption of payment channels.** XUD acts as a wrapper and encapsulates lightning and raiden daemons with as little modifications as possible.
+Each participating exchange runs its own XUD instance, which opens payment channels on each chain it desires to enable trading for. This means, if for example an exchange wants to enable its users to trade bitcoin and litecoin through XUD, it has to open a payment channel on the litecoin blockchain & on the bitcoin blockchain to at least one other exchange in the union. Payment channels in XU are meant to be kept open long-term. Since routing is and will be relatively unreliable for larger amounts in the foreseeable future, the current design encourages larger, direct channels between exchanges. Currently, lnd's maximum channel size is temporarily limited to 0.16777216 BTC. This can be lifted using a so-called [wumbo](https://github.com/lightningnetwork/lightning-rfc/wiki/Lightning-Specification-1.1-Proposal-States)-bit. To achieve long-term scalability, even with routing, we will rely on multiple channels, which carry out partial transfers of one atomic payment as proposed in the [AMP protocol by Laolu and Conner](https://lists.linuxfoundation.org/pipermail/lightning-dev/2018-February/000993.html).
```

Comment:
Second sentence can say "For example, if an exchange wants to enable trading on bitcoin and litecoin through xud, it would need payment channels on both the litecoin and bitcoin blockchains."

And regarding AMP I think it'd be better to say that xud will "take advantage of" AMP rather than "rely" on it since it's definitely not required.

Reply:
Fixed